tests/group_vars/all.yml

```diff
@@ -0,0 +1,2 @@
+namespace: my-ripsaw
+ripsaw_dir: /home/dry/working/ripsaw
```

Comment:
Does this need to be changed?

Reply:
It should be doc'd most likely. That dir gets overwritten with the current working directory when run via the test.sh script.

Reply:
Are you saying there are docs updated somewhere in this PR that I'm not seeing, or are you acknowledging that docs need to be done? Stupid english--

Reply:
Acknowledging that docs need to be done for this change.

Reply:
@dustinblack please check out the updated docs